### DISCUSSION TITLE
Email notification when charging stopped and charge isn't 100%

### DISCUSSION THREAD

**Discussion by David42fr** [teslamate-org/teslamate#2947] [Ideas]
Hello there,

I encounter a lot of problem with electricity quality ans charging stopped once a day or more.

Is there a way to receive a notification by mail telling us that charging stopped and the percentage is xx%? 

Thanks

**Comment by natrlhy** [comment]
The Tesla App does this

**Comment by David42fr** [comment]
No, it sends a notification on the phone, not an email ;)

**Comment by DrMichael** [comment]
You could do it with NodeRed and Mosquito... Something like this:
![image](https://user-images.githubusercontent.com/8087682/195978800-495de3fd-1bbb-41c0-84cc-95673bc740e2.png)
